src/plugins/immutable-properties.ts

```diff
@@ -145,11 +145,20 @@ function propertyFromAttribute(attribute:ValueObject.Attribute):ObjC.Property {
 }
 
 function isImportRequiredForAttribute(typeLookups:ObjectGeneration.TypeLookup[], attribute:ValueObject.Attribute):boolean {
-  return ObjCImportUtils.shouldIncludeImportForType(typeLookups, attribute.type.name);
+  return Maybe.match(function (protocol) {
+    return ObjCImportUtils.shouldIncludeImportForType(typeLookups, attribute.type.name) ||
+      ObjCImportUtils.shouldIncludeImportForType(typeLookups, protocol);
+  }, function () {
+    return ObjCImportUtils.shouldIncludeImportForType(typeLookups, attribute.type.name);
```

Comment:
Store the results of this to a const variable outside the match, it's duplicated in the two cases